packages/core/useWakeLock/index.ts

```diff
@@ -0,0 +1,60 @@
+import { ref } from 'vue-demi'
+import { useEventListener } from '../useEventListener'
+import { defaultNavigator, ConfigurableNavigator } from '../_configurable'
+
+type WakeLockType = 'screen'
+
+export interface WakeLockSentinel extends EventTarget {
+  type: WakeLockType
+  released: boolean
+  release: () => Promise<void>
+}
+
+type NavigatorWithWakeLock = Navigator & {
+  wakeLock: { request: (type: WakeLockType) => Promise<WakeLockSentinel> }
+}
+
+/**
+ * Reactive Screen Wake Lock API.
+ *
+ * @see https://vueuse.org/useWakeLock
+ * @param options
+ */
+export const useWakeLock = ({ navigator = defaultNavigator }: ConfigurableNavigator = {}) => {
+  let wakeLock: WakeLockSentinel | null
+  const isSupported = navigator && 'wakeLock' in navigator
+  const isActive = ref(false)
+
+  async function onVisibilityChange() {
+    if (!isSupported || !wakeLock) return
+
+    if (document && document.visibilityState === 'visible')
+      wakeLock = await (navigator as NavigatorWithWakeLock).wakeLock.request('screen')
+
+    isActive.value = !wakeLock.released
+  }
+
+  useEventListener(document, 'visibilitychange', onVisibilityChange, { passive: true })
```

Comment:
We will need to have `document` configurable as well.

Reply:
Thanks for letting me know!
I fixed it, so could you please confirm it again?